recipe/meta.yaml

```diff
@@ -3,7 +3,8 @@
 # Copyright (c) 2017, Continuum Analytics, Inc. All rights reserved.
 {% set version = "11.0.3" %}
 {% set version_patch = "450.51.06" %}  # [linux64]
-{% set version_patch = "450.51.06" %}  # [linux-ppc64le]
+{% set version_patch = "450.51.06" %}  # [aarch64]
```

Comment:
```suggestion
```

Reply:
This would have to be a manual change later on, so I think it is reasonable to leave it in to make less work in the future when ARM is enabled

Reply:
Yeah was just thinking we could keep this as just Power to keep things simple here as we already have those changes in PR ( https://github.com/conda-forge/cudatoolkit-feedstock/pull/4 ). Though I guess it doesn't matter much either way.

Reply:
Yeah, it is more work to remove, but if you want to keep this pure Power 8, I can remove

Reply:
Yeah that's why I included the changes here to hopefully make that easier.